cogs/rosterize.py

```diff
@@ -230,6 +244,8 @@ def rosterdetail(self, c, sid, rname=None):
             await self.bot.say("Roster name too long")
         elif not self.intable(c, message.server.id, margs[1]):
             await self.bot.say(margs[1] + " does not exist")
+        elif self.checkOwner(c, message.server.id, margs[1], message.author.id) == False:
```

Comment:
change to 'not' instead of comparing to False; change to snake_case and rename function name to be more bool-readable